extract/extract/extractors/otto_de.py

```diff
@@ -34,26 +35,34 @@ def extract_otto_de(parsed_page: ParsedPage) -> Optional[Product]:
     microdata = safely_return_first_element(parsed_page.schema_org.get(MICRODATA, [{}]))
     properties = microdata.get("properties", {})
 
-    name = properties.get("name", None)
-    brand = properties.get("brand", None)
+    name = properties.get("name")
```

Comment:
What's the default return value when not specifying it here? Would the get request fail?

Reply:
The default is `None`, so it is not necessary to specify it. If the request fails, it just returns `None`.